### DISCUSSION TITLE
Improving Version Display in npx @astrojs/upgrade Command

### DISCUSSION THREAD

**Discussion by outslept** [withastro/roadmap#1069] [Proposal]
### Body

# Summary

Currently, npx @astrojs/upgrade command shows only the target version like `@astrojs/tailwind will be updated to v5.1.3.` This RFC proposes to enhance the output by showing both current and target versions for better visibility

# Background & Motivation

The current output format lacks clarity in showing version changes. Adding a simple "from -> to" format would improve visibility and DX when updating Astro packages.

# Goals

- Make the upgrade process "more transparent"
- Simplify version change tracking
- Improve DX during updates

# Example

Current output:

```
@astrojs/tailwind will be updated to v5.1.3
```

Proposed output (or mb some variation of this):

```
@astrojs/tailwind: v.x -> v5.1.3
```

Or we can just remain the same style and do it like this:

```
@astrojs/tailwind will be updated: v.x -> v5.1.3
```

**Comment by delucis** [comment]
This is a good idea I think. I think we’d be happy to receive a PR for this if you wanted to tackle it — no need for a small thing like this to go through the RFC process.

**Comment by florian-lefebvre** [comment]
This was implemented
